src/Registration/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+#========================================================================
+# Author: Richard Brown
+# Copyright 2016, 2017 University College London
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#=========================================================================
+
+# To be able to build with niftyreg, you need to append the cmake_cxx_flags
+# to suppress errors from converting float to double
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-narrowing")
+
+# SIRF stuff
```

Comment:
shouldn't need any of this now it's an integral part of SIRF? Certainly not the `find_package` things